composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "os2forms/os2forms_organisation_api",
+    "name": "itk-dev/serviceplatformen_organisation_api",
```

Comment:
```suggestion
    "name": "itk-dev/serviceplatformen-organisation-api",
```
(cf. https://getcomposer.org/doc/02-libraries.md#every-project-is-a-package: “… the convention is to use dashes for word separation.”)

Reply:
Updated to align with convention.